jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/web/WebScreenshotCreator.java

```diff
@@ -25,24 +25,34 @@
 @RequiredArgsConstructor
 public class WebScreenshotCreator {
 
+    public enum type {FAILED, DEBUG, MANUAL, WAIT, STEP}
```

Comment:
Minor: enums should use class conventions for naming - e.g. first capital letter `Type`

Reply:
Sorry fixed.